dev/how-to/troubleshoot/tidb-in-kubernetes.md

```diff
@@ -0,0 +1,268 @@
+
+    Use the `ping` command to diagnose whether the three layers of network to the destination IP are connected (the destination IP is the ClusterIP resolved using `dig`):
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    ping ${TARGET_IP}
+    ```
+
+    - If the `ping` check fails, refer to [Debugging Kubernetes Networking](https://www.praqma.com/stories/debugging-kubernetes-networking/) for troubleshooting.
+
+    - If the `ping` check succeeds, continue to check whether the target port is open by using `telnet`:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        telnet ${TARGET_IP} ${TARGET_PORT}
+        ```
+
+        If the `telnet` check fails, check whether the port corresponding to the Pod is correctly exposed and whether the applied port is correctly configured:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        # Checks whether the ports are consistent.
+        kubectl -n ${namespace} get po ${pod_name} -ojson | jq '.spec.containers[].ports[].containerPort'
+
+        # Checks whether the application is correctly configured to serve the specified port.
+        # The default port of PD is 2379 when not configured.
+        kubectl -n ${namespace} -it exec ${pod_name} -- cat /etc/pd/pd.toml | grep client-urls
+        # The default port of PD is 20160 when not configured.
+        kubectl -n ${namespace} -it exec ${pod_name} -- cat /etc/tikv/tikv.toml | grep addr
+        # The default port of TiDB is 4000 when not configured.
+        kubectl -n ${namespace} -it exec ${pod_name} -- cat /etc/tidb/tidb.toml | grep port
+        ```
+
+## The Pod is in the Pending state
+
+The Pending state of a Pod is usually caused by insufficient resources, such as the following conditions:
+
+- The `StorageClass` of the PVC used by PD, TiKV, Monitor Pod does not exist or the PV is insufficient.
+- No nodes in the Kubernetes cluster can satisfy the CPU or memory applied by the Pod
+
+You can check the specific reason for Pending by using the `kubectl describe pod` command:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl describe po -n ${namespace} ${pod_name}
+```
+
+- If the CPU or memory resources are insufficient, you can lower the CPU or memory resources applied by the corresponding component, or add a new Kubernetes node.
+
+- If the `StorageClass` of the PVC cannot be found, delete the TiDB Pod and the corresponding PVC. Then, in the `values.yaml` file, change `StorageClassName` to the name of the `StorageClass` available in the cluster. Run the following command to get the `StorageClass` available in the cluster:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl get storageclass
+    ```
+
+- If a `StorageClass` exists in the cluster but the available PV is insufficient, you need to add the corresponding PV resources. For Local PV, you can expand it by referring to [Local PV Configuration](/reference/configuration/tidb-in-kubernetes/local-pv-configuration.md).
```

Comment:
```suggestion
- If a `StorageClass` exists in the cluster but the available PV is insufficient, you need to add PV resources correspondingly. For Local PV, you can expand it by referring to [Local PV Configuration](/reference/configuration/tidb-in-kubernetes/local-pv-configuration.md).
```

Reply:
Cannot find the reference object in the context by saying "the corresponding PV resources".
But we can use the adv for the verb "add".